charts/camunda-platform-8.8/values.yaml

```diff
@@ -142,6 +142,8 @@ global:
     app: camunda-platform
   ## @param global.commonLabels can be used to apply mutable labels to all Camunda resources.
   commonLabels: {}
+  ## @param global.nodeSelector can be used to define on which nodes all Camunda pods should run, unless overridden by component-specific nodeSelector
```

Comment:
To avoid confusion, we clarify that it's K8s nodes (not Zeebe nodes or anything similar).

```
can be used to define Kubernetes node selectors to run Camunda components, unless overridden by component-specific nodeSelector
```